additional_modules/additional_modules_fastq_thresholding.py

```diff
@@ -0,0 +1,31 @@
+def gc_content(seq: str):
+    gc_total = 0
+    for nucleotide in seq:
+        if nucleotide.lower() == 'g' or nucleotide.lower() == 'c':
+            gc_total += 1
+    return gc_total*100/len(seq)
+
+
+def is_in_gc_bounds(seq: str, gc_lower_bound: int, gc_upper_bound: int):
+    if gc_lower_bound <= gc_content(seq) <= gc_upper_bound:
+        return True
+    else:
+        return False
```

Comment:
Можно сделать без else, потому что если код зайдет в if, то функция вернет True и закончит свою работу. Если не зайдет, то дойдет до следующий строки и вернет False.
```suggestion
    return False
```